src/main/java/uk/gov/hmcts/reform/sscs/service/NotificationUtils.java

```diff
@@ -162,11 +159,18 @@ public static final Hearing getLatestHearing(SscsCaseData sscsCaseData) {
         Comparator<Hearing> compareByIdAndDate = new Comparator<Hearing>() {
             @Override
             public int compare(Hearing o1, Hearing o2) {
-                Integer idCompare = o1.getValue().getHearingId().compareTo(o2.getValue().getHearingId());
+                HearingDetails hearingDetails = o1.getValue();
+                HearingDetails nextHearingDetails = o2.getValue();
+                Integer idCompare = 0;
+
+                if (!StringUtils.isEmpty(hearingDetails.getHearingId()) && !StringUtils.isEmpty(nextHearingDetails.getHearingId())) {
+                    idCompare = hearingDetails.getHearingId().compareTo(nextHearingDetails.getHearingId());
```

Comment:
I'd probably use https://commons.apache.org/proper/commons-lang/javadocs/api-release/org/apache/commons/lang3/builder/CompareToBuilder.html to make this clearer.